scopes/generator/generator/templates/component-generator/index.ts

```diff
@@ -1,31 +1,41 @@
 import { ComponentContext, ComponentTemplate } from '@teambit/generator';
 import { indexFile } from './files/index';
-import { aspectFile } from './files/aspect-file';
-import { docsFile } from './files/docs-file';
```

Comment:
What about docs here?

Reply:
@GiladShoham I followed the pattern we have in our component generator for React